src/components/Wizard/Wizard.tsx

```diff
@@ -112,7 +112,12 @@ export default function Wizard<T>({
 	}
 
 	return (
-		<Modal id="dms-upload-wizard" open={true} onClose={() => terminate()} closeOnBackdropClick={false}>
+		<Modal
+			id="dms-upload-wizard"
+			open={true}
```

Comment:
I realize this might not be your doing but `open` instead of `open={true}` is preferred